pkg/webhook/server/server.go

```diff
@@ -102,39 +103,19 @@ func (s *Server) checkTLS() error {
 		return errors.Wrap(err, "failed reading for TLS key")
 	}
 
-	_, err = triple.ParsePrivateKeyPEM(key)
-	if err != nil {
-		return errors.Wrap(err, "failed parsing TLS key")
-	}
-
 	certPEM, err := ioutil.ReadFile(path.Join(s.webhookServer.CertDir, corev1.TLSCertKey))
 	if err != nil {
 		return errors.Wrap(err, "failed reading for TLS cert")
 	}
 
-	certs, err := triple.ParseCertsPEM(certPEM)
-	if err != nil {
-		return errors.Wrap(err, "failed parsing TLS cert")
-	}
-
 	caPEM, err := s.certManager.CABundle()
 	if err != nil {
 		return errors.Wrap(err, "failed to retrieve CA cert")
 	}
 
-	cas := x509.NewCertPool()
-	ok := cas.AppendCertsFromPEM([]byte(caPEM))
-	if !ok {
-		return errors.New("failed to parse CA certificate")
-	}
-
-	opts := x509.VerifyOptions{
-		Roots:   cas,
-		DNSName: certs[0].DNSNames[0],
-	}
-
-	if _, err := certs[0].Verify(opts); err != nil {
-		return errors.Wrap(err, "failed to verify certificate")
+	err = triple.VerifyTLS(certPEM, key, caPEM)
+	if err != nil {
+		return errors.Wrapf(err, "failed verifying %s/%s", certsPath, keyPath)
 	}
 
 	return nil
```

Comment:
since in the logs we only see the negative logs like `E0601 09:25:57.673216       1 server.go:128] failed verifying /etc/webhook/certs/tls.crt//etc/webhook/certs/tls.key: failed parsing PEM TLS key: data does not contain a valid RSA or ECDSA private key` I think we should also add a positive log so that the user/developer will understand that the secret is good now.
maybe inside `VerifyTLS()`? what do you think?